lib/api/core/httpRouter/index.js

```diff
@@ -68,7 +68,6 @@ class Router {
 
     // Add an automatic HEAD route on the '/' url, answering with default headers
     attach('/', (request, cb) => {
-      Object.assign(request.input.args, this.defaultHeaders);
```

Comment:
Default headers are automatically assigned to the `RequestResponse` object, and they serve no purpose in the input part of the request, so this line is actually useless.